reconcile/templating/renderer.py

```diff
@@ -267,8 +267,8 @@ def reconcile(
                     if output:
                         outputs.append(output)
 
-                if not dry_run:
-                    p.write(outputs)
+                    if not dry_run:
+                        p.write(outputs)
```

Comment:
`outputs` is defined outside the loop, so this will write it multiple times like `[1]`, `[1,2]`, `[1,2,3]`, any issues with write the whole `outputs` once?

Reply:
It doesn't matter, cause it's iterating. That's why it worked in manual test anyhow.
But great catch, the list can go away and `output` can be written directly.

I will add tests later today :+1: